src/components/todoForm/TodoForm.tsx

```diff
@@ -24,6 +27,17 @@ const TodoForm: React.FC = () => {
     setTitle("");
   };
 
+  const handleAddTodos = async (todos: ITodo[]) => {
+    const promises = todos.map((todo) => addTodoMutation(todo));
+    await Promise.all(promises);
+
+    queryClient.invalidateQueries("todos");
```

Comment:
create seperate mutation for sending todos list in one request

Reply:
I asked you about it and I haven't find an answer that's why I did it in that way. And I don't how to do in another way

Reply:
https://stackoverflow.com/questions/57868537/post-collection-of-objects-in-json-server try this